src/mesh/difops.cxx

```diff
@@ -324,65 +326,15 @@ const Field3D Vpar_Grad_par_LCtoC(const Field3D &v, const Field3D &f, REGION reg
       result[i] -= (vval.p >= 0.0) ? vval.p * fval.c : vval.p * fval.p;
     }
   }
-  else if (vUseUpDown) {
-    // Only v has up/down fields
-    // f must shift to field aligned coordinates
-    Field3D f_fa = mesh->toFieldAligned(f);
```

Comment:
We're shifting `f` to be field aligned such that it should make sense to multiply with `v`, which has field aligned data available (in yup and ydown), so I think this is ok but could be missing something.

Reply:
I may well have misunderstood. I thought that to/fromFieldAligned rotate the points in the z-direction by zShift, so the toroidal position of the point (i,j,k) would be different for field-aligned and non-field-aligned fields?